### ISSUE TITLE
This extension failed to redirect a network request to <base64> because another extension (Disconnect) redirected it to about:blank.

### ISSUE THREAD

**Issue by geekpete** [gorhill/uBlock#1965]
Please skip reviewing this one until I can provide you with solid reproducible test....
But if this is a known issue then disregard and close.

### Describe the issue
Possible compatibility issue with disconnect and ublock origin?

Also have the following extensions installed:
Disconnect 5.18.23
Disconnect Privacy Icons 1.0.23

Error seen in the Extensions section:

Warning:
This extension failed to redirect a network request to data:application/javascript;base64,KGZ1bmN0aW9uKCkgewoJdmFyIHA7CgkvLyBodHRwczovL2RldmVsb3BlcnMuZ29vZ2xlLmNvbS9kb3VibGVjbGljay1ncHQvcmVmZXJlbmNlCgl2YXIgbm9vcGZuID0gZnVuY3Rpb24oKSB7CgkJOwoJfTsKCXZhciBub29wdGhpc2ZuID0gZnVuY3Rpb24oKSB7CgkJcmV0dXJuIHRoaXM7Cgl9OwoJdmFyIG5vb3BudWxsZm4gPSBmdW5jdGlvbigpIHsKCQlyZXR1cm4gbnVsbDsKCX07Cgl2YXIgbm9vcGFycmF5Zm4gPSBmdW5jdGlvbigpIHsKCQlyZXR1cm4gW107Cgl9OwoJdmFyIG5vb3BzdHJmbiA9IGZ1bmN0aW9uKCkgewoJCXJldHVybiAnJzsKCX07CgkvLwoJdmFyIGNvbXBhbmlvbkFkc1NlcnZpY2UgPSB7CgkJYWRkRXZlbnRMaXN0ZW5lcjogbm9vcHRoaXNmbiwKCQllbmFibGVTeW5jTG9hZGluZzogbm9vcGZuLAoJCXNldFJlZnJlc2hVbmZpbGxlZFNsb3RzOiBub29wZm4KCX07Cgl2YXIgY29udGVudFNlcnZpY2UgPSB7CgkJYWRkRXZlbnRMaXN0ZW5lcjogbm9vcHRoaXNmbiwKCQlzZXRDb250ZW50OiBub29wZm4KCX07Cgl2YXIgUGFzc2JhY2tTbG90ID0gZnVuY3Rpb24oKSB7CgkJOwoJfTsKCXAgPSBQYXNzYmFja1Nsb3QucHJvdG90eXBlOwoJcC5kaXNwbGF5ID0gbm9vcGZuOwoJcC5nZXQgPSBub29wbnVsbGZuOwoJcC5zZXQgPSBub29wdGhpc2ZuOwoJcC5zZXRDbGlja1VybCA9IG5vb3B0aGlzZm47CglwLnNldFRhZ0ZvckNoaWxkRGlyZWN0ZWRUcmVhdG1lbnQgPSBub29wdGhpc2ZuOwoJcC5zZXRUYXJnZXRpbmcgPSBub29wdGhpc2ZuOwoJcC51cGRhdGVUYXJnZXRpbmdGcm9tTWFwID0gbm9vcHRoaXNmbjsKCXZhciBwdWJBZHNTZXJ2aWNlID0gewoJCWFkZEV2ZW50TGlzdGVuZXI6IG5vb3B0aGlzZm4sCgkJY2xlYXI6IG5vb3BmbiwKCQljbGVhckNhdGVnb3J5RXhjbHVzaW9uczogbm9vcHRoaXNmbiwKCQljbGVhclRhZ0ZvckNoaWxkRGlyZWN0ZWRUcmVhdG1lbnQ6IG5vb3B0aGlzZm4sCgkJY2xlYXJUYXJnZXRpbmc6IG5vb3B0aGlzZm4sCgkJY29sbGFwc2VFbXB0eURpdnM6IG5vb3BmbiwKCQlkZWZpbmVPdXRPZlBhZ2VQYXNzYmFjazogZnVuY3Rpb24oKSB7IHJldHVybiBuZXcgUGFzc2JhY2tTbG90KCk7IH0sCgkJZGVmaW5lUGFzc2JhY2s6IGZ1bmN0aW9uKCkgeyByZXR1cm4gbmV3IFBhc3NiYWNrU2xvdCgpOyB9LAoJCWRpc2FibGVJbml0aWFsTG9hZDogbm9vcGZuLAoJCWRpc3BsYXk6IG5vb3BmbiwKCQllbmFibGVBc3luY1JlbmRlcmluZzogbm9vcGZuLAoJCWVuYWJsZVNpbmdsZVJlcXVlc3Q6IG5vb3BmbiwKCQllbmFibGVTeW5jUmVuZGVyaW5nOiBub29wZm4sCgkJZW5hYmxlVmlkZW9BZHM6IG5vb3BmbiwKCQlnZXQ6IG5vb3BudWxsZm4sCgkJZ2V0QXR0cmlidXRlS2V5czogbm9vcGFycmF5Zm4sCgkJcmVmcmVzaDogbm9vcGZuLAoJCXNldDogbm9vcHRoaXNmbiwKCQlzZXRDYXRlZ29yeUV4Y2x1c2lvbjogbm9vcHRoaXNmbiwKCQlzZXRDZW50ZXJpbmc6IG5vb3BmbiwKCQlzZXRDb29raWVPcHRpb25zOiBub29wdGhpc2ZuLAoJCXNldExvY2F0aW9uOiBub29wdGhpc2ZuLAoJCXNldFB1Ymxpc2hlclByb3ZpZGVkSWQ6IG5vb3B0aGlzZm4sCgkJc2V0VGFnRm9yQ2hpbGREaXJlY3RlZFRyZWF0bWVudDogbm9vcHRoaXNmbiwKCQlzZXRUYXJnZXRpbmc6IG5vb3B0aGlzZm4sCgkJc2V0VmlkZW9Db250ZW50OiBub29wdGhpc2ZuLAoJCXVwZGF0ZUNvcnJlbGF0b3I6IG5vb3BmbgoJfTsKCXZhciBTaXplTWFwcGluZ0J1aWxkZXIgPSBmdW5jdGlvbigpIHsKCQk7Cgl9OwoJcCA9IFNpemVNYXBwaW5nQnVpbGRlci5wcm90b3R5cGU7CglwLmFkZFNpemUgPSBub29wdGhpc2ZuOwoJcC5idWlsZCA9IG5vb3BudWxsZm47Cgl2YXIgU2xvdCA9IGZ1bmN0aW9uKCkgewoJCTsKCX07CglwID0gU2xvdC5wcm90b3R5cGU7CglwLmFkZFNlcnZpY2UgPSBub29wdGhpc2ZuOwoJcC5jbGVhckNhdGVnb3J5RXhjbHVzaW9ucyA9IG5vb3B0aGlzZm47CglwLmNsZWFyVGFyZ2V0aW5nID0gbm9vcHRoaXNmbjsKCXAuZGVmaW5lU2l6ZU1hcHBpbmcgPSBub29wdGhpc2ZuOwoJcC5nZXQgPSBub29wbnVsbGZuOwoJcC5nZXRBZFVuaXRQYXRoID0gbm9vcGFycmF5Zm47CglwLmdldEF0dHJpYnV0ZUtleXMgPSBub29wYXJyYXlmbjsKCXAuZ2V0Q2F0ZWdvcnlFeGNsdXNpb25zID0gbm9vcGFycmF5Zm47CglwLmdldERvbUlkID0gbm9vcHN0cmZuOwoJcC5nZXRTbG90RWxlbWVudElkID0gbm9vcHN0cmZuOwoJcC5nZXRTbG90SWQgPSBub29wdGhpc2ZuOwoJcC5nZXRUYXJnZXRpbmcgPSBub29wYXJyYXlmbjsKCXAuZ2V0VGFyZ2V0aW5nS2V5cyA9IG5vb3BhcnJheWZuOwoJcC5zZXQgPSBub29wdGhpc2ZuOwoJcC5zZXRDYXRlZ29yeUV4Y2x1c2lvbiA9IG5vb3B0aGlzZm47CglwLnNldENsaWNrVXJsID0gbm9vcHRoaXNmbjsKCXAuc2V0Q29sbGFwc2VFbXB0eURpdiA9IG5vb3B0aGlzZm47CglwLnNldFRhcmdldGluZyA9IG5vb3B0aGlzZm47CgkvLwoJdmFyIGdwdCA9IHdpbmRvdy5nb29nbGV0YWcgfHwge307Cgl2YXIgY21kID0gZ3B0LmNtZCB8fCBbXTsKCWdwdC5hcGlSZWFkeSA9IHRydWU7CglncHQuY21kID0gW107CglncHQuY21kLnB1c2ggPSBmdW5jdGlvbihhKSB7CgkJdHJ5IHsKCQkJYSgpOwoJCX0gY2F0Y2ggKGV4KSB7CgkJfQoJCXJldHVybiAxOwoJfTsKCWdwdC5jb21wYW5pb25BZHMgPSBmdW5jdGlvbigpIHsgcmV0dXJuIGNvbXBhbmlvbkFkc1NlcnZpY2U7IH07CglncHQuY29udGVudCA9IGZ1bmN0aW9uKCkgeyByZXR1cm4gY29udGVudFNlcnZpY2U7IH07CglncHQuZGVmaW5lT3V0T2ZQYWdlU2xvdCA9IGZ1bmN0aW9uKCkgeyByZXR1cm4gbmV3IFNsb3QoKTsgfTsKCWdwdC5kZWZpbmVTbG90ID0gZnVuY3Rpb24oKSB7IHJldHVybiBuZXcgU2xvdCgpOyB9OwoJZ3B0LmRpc2FibGVQdWJsaXNoZXJDb25zb2xlID0gbm9vcGZuOwoJZ3B0LmRpc3BsYXkgPSBub29wZm47CglncHQuZW5hYmxlU2VydmljZXMgPSBub29wZm47CglncHQuZ2V0VmVyc2lvbiA9IG5vb3BzdHJmbjsKCWdwdC5wdWJhZHMgPSBmdW5jdGlvbigpIHsgcmV0dXJuIHB1YkFkc1NlcnZpY2U7IH07CglncHQucHViYWRzUmVhZHkgPSB0cnVlOwoJZ3B0LnNpemVNYXBwaW5nID0gZnVuY3Rpb24oKSB7IHJldHVybiBuZXcgU2l6ZU1hcHBpbmdCdWlsZGVyKCk7IH07Cgl3aW5kb3cuZ29vZ2xldGFnID0gZ3B0OwoJd2hpbGUgKCBjbWQubGVuZ3RoICE9PSAwICkgewoJCWdwdC5jbWQucHVzaChjbWQuc2hpZnQoKSk7Cgl9Cn0pKCk7 because another extension (Disconnect) redirected it to about:blank.

Flip this into https://www.base64decode.org/ to decode,etc.

But essentially it seems that this was blocked by Disconnect: "https://developers.google.com/analytics/devguides/collection/analyticsjs/"

Are you using google analytics inside ublock for tracking?
Why was ublock origin redirecting a request to/from this google analytics url?
Just trying to understand how it works.

### One or more specific URLs where the issue occurs

Ublock was prevented from redirecting https://developers.google.com/analytics/devguides/collection/analyticsjs/
Unsure which specific url triggers it, will test more and report back.

### Screenshot in which the issue can be seen

[![Screen Shot 2016-09-07 at 9.08.20 AM.png](https://s13.postimg.org/dxnomh12f/Screen_Shot_2016_09_07_at_9_08_20_AM.png)](https://postimg.org/image/g281nk2oz/)
### Steps for anyone to reproduce the issue

1. Be as detailed as possible:
1. Because we are not looking over your shoulder and
1. Keep in mind: Nobody can read your mind.

Will follow up with more detail on how to replicate after more testing to identify any specific urls triggering this.

### Your settings

[If you fail to provide this info, I will mark the issue as invalid. Lists all settings which differs from default settings]

- Browser/version: 
Chromium	51.0.2683.0 (Developer Build) (64-bit)
Revision	dba3b6817d37e512d4722aa642353b7d48db1a7d-refs/heads/master@{#381909}
OS	Mac OS X 
Blink	537.36 (@dba3b6817d37e512d4722aa642353b7d48db1a7d)
JavaScript	V8 5.1.168
Flash	(Disabled)
User Agent	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2683.0 Safari/537.36
Command Line	/Applications/Chromium.app/Contents/MacOS/Chromium --enable-avfoundation --flag-switches-begin --flag-switches-end
Executable Path	/Applications/Chromium.app/Contents/MacOS/Chromium
Profile Path	/Users/peter/Library/Application Support/Chromium/Default
Variations	16e0dd70-3f4a17df
b3888d8d-ac4538a3
cf5ed6e1-3f4a17df
775ebbd7-3f4a17df
31362330-3f4a17df
dd4da2fc-3f4a17df
a3903585-b1b2c74f
64cbdfc2-3f4a17df
f5dd6118-2f5721af
4ea303a6-3f4a17df
4117e878-23622c9b
dbffab5d-f51b51
30e679f-3f4a17df
ad6d27cc-3e870323
c8b9b12d-d93a0620
867c4c68-3f4a17df
c674d85e-3f4a17df
a70306ec-3f4a17df
b0dc61a1-3f4a17df

- uBlock Origin version: 
 1.9.4

##### Your filter lists
All ticked,and DEU in the language specific ones ticked.

**Comment by gwarser** [comment]
https://github.com/gorhill/uBlock/issues/1170#issuecomment-168989068

**Comment by geekpete** [comment]
Thanks.

I'm with @gorhill on users controlling their own browsing experience. I'll be the judge of what my browser sends back to server sites and the more tools and methods I have to control and make this transparent the better.

**Comment by gorhill** [comment]
Unfortunately there is no solution on my side for this one, [as per Chrome API](https://developer.chrome.com/extensions/webRequest#implementation):

> An extension is not notified if its instruction to modify or redirect has been ignored

uBO's logger will tell you what is happening internally, it will also show you redirects (or attempts at). If the harmless warning annoys you, you can choose to whitelist in uBO whatever is being redirected so that the other extension will be free to handle the network request with no conflict.

**Comment by gorhill** [comment]
Duplicate of #1170.
